test/compilable/test_ctfeonly.d

```diff
@@ -0,0 +1,42 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -betterC
+// POST_SCRIPT: compilable/extra-files/ctfeonly-postscript.sh
+// I guess the platforms below don't have nm
+// DISABLED: win32 win64 osx
```

Comment:
OSX does, and for windows you'll need dumpbin: https://docs.microsoft.com/en-us/cpp/build/reference/dumpbin-reference?view=vs-2019

Reply:
OK, let's see whether I can get this to work on windows. Do you know how to detect the OS in postscripts?

Reply:
You can use the OS variable, see test/Readme.md